test/tests.py

```diff
@@ -297,7 +297,6 @@ def _():
     )
     # Backup should include important files
     files = {
-        "bitcoind": "var/lib/bitcoind/wallet.dat",
```

Comment:
Shouldn't we still test that the wallet created by `$cli createwallet "test"` is being backed up?

Reply:
Originally, I wanted avoid adding the costly `backups` test to the `regtest` scenario. But there's actually a much simpler and faster solution. Done via [force push](https://github.com/fort-nix/nix-bitcoin/compare/506c1cd066949c40ec26f722dcb31cd3e3b320dd..b114d0c3b129f64ea58ba9c97c7fb91abe3c2057).